data/json/flags.json

```diff
@@ -1136,8 +1136,21 @@
   },
   {
     "id": "COLLAPSIBLE_STOCK",
+    "//": "this is deprecated for the COLLAPSED_STOCK and FOLDED_STOCK, kept around for mod support",
     "type": "json_flag"
   },
+  {
+    "id": "FOLDED_STOCK",
+    "//": "for weapons that stock folds to one side, no change in volume.",
+    "type": "json_flag",
+    "info": "This items stock is folded it will be hard to shoot with."
```

Comment:
items → item's

Reply:
Also missing comma before "it"

Reply:
I'd also give <info> tag as well while at it.

Reply:
Finally (sorry about the comment streak); "item" or "gun"? I think there's no other items that can have stocks.